.github/workflows/ruby.yml

```diff
@@ -0,0 +1,27 @@
+name: CI
+
+on:
+  push:
+    branches: [ master ]
+  pull_request:
+    branches: [ master ]
+
+jobs:
+  test:
+    runs-on: ubuntu-latest
+    services:
+      redis:
+        image: redis:alpine
+        ports: ["6379:6379"]
+    steps:
+    - uses: actions/checkout@v2
+    - name: Set up Ruby
+      uses: ruby/setup-ruby@v1
+      with:
+        ruby-version: 2.6.6
+    - name: Install dependencies
+      run: bundle install
+    - name: Lint with RuboCop
+      run: bundle exec rubocop --parallel
+    - name: Run tests
+      run: bundle exec rake spec
```

Comment:
Should we add Danger?

Reply:
We should indeed!

Reply:
You are right about the travisCI stuff. Will be running the checks in parallel for a bit, after we make sure everything is OK will be sending another PR removing travisCI.